modules/core/core-api/src/main/java/com/enonic/xp/content/ContentVersion.java

```diff
@@ -76,14 +76,34 @@ public static Builder create()
     }
 
     @Override
-    public int compareTo( final ContentVersion o )
+    public int compareTo( final ContentVersion that )
```

Comment:
Leave alone this broken implementation (Deprecate?). Write a separate Comparator. Also See #7440